behavior/microservice/templates/base.html

```diff
@@ -0,0 +1,19 @@
+<!doctype html>
+<html>
+  <head>
+    <title>{{ title }}</title>
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.1/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-+0n0xVW2eSR5OomGNYDnhzAbDsOXxcvSN1TPprVMTNDbiYZCxYbOOl7+AMvyTG2x" crossorigin="anonymous">
```

Comment:
I have mixed opinions on cdn vs just downloading the css and including a minified version. Do you have strong opinions against downloading these to `microservice/static`? I'm thinking about the no-internet airplane workflow right now.

Reply:
Switched to downloading these to `microservice/static`.